data/hai/hai missions.txt

```diff
@@ -35,14 +38,105 @@ mission "First Contact: Hai"
 			
 			`	The human merchant laughs. "Probably because most of us came here to escape from the chaos and fighting in human space. The last thing we want is for it to follow us here. Not that I'm saying you can't tell anyone about the wormhole, but if I were you I wouldn't go spreading the news too widely either. And take a look around Hai space before you leave; you'll find that we could learn a lot from them."`
 			`	"So I can travel anywhere I want in your territory?" you ask the Hai.`
-			
+
 			label travel
 			`	"Yes," it says, "but be careful. The north is the territory of some renegade Hai: bandits and pirates, living off what they steal from the rest of us. If you travel among them, you will not be safe."`
 			`	You talk for a while longer, but do not gain much additional information except that several of their worlds are willing to sell Hai technology even to human beings. You thank them both for stopping to talk with you, and the Hai responds, "We are always glad to welcome new friends. May peace dwell in your heart until we meet again."`
 				decline
+			
+			label unfettered
+			
+			`	The Hai here seem more friendly and approachable than those you met earlier. However, the most surprising aspect of the spaceport is not the aliens, but the fact that human merchants and other civilians are walking among them and are clearly at home here. Would you like to find someone who can tell you what is going on here?`
+			choice
+				`	(Sure.)`
+				`	(Not right now.)`
+					defer
+					
+			`	You walk up to a human merchant who is busy haggling with a Hai. They interrupt their conversation when they see you approaching, and the human gestures for you to join them. "First time down the rabbit hole?" he asks.`
+			`	You shake your head and explain how you met with the Unfettered that called themselves the 'True Hai'.`
+			`	The hai says, "Ah, you have met our wayward sisters and brothers to the north. Is that what you have come to ask us about?`
+			choice
+				`	"Yes," you say. "Can you tell me why you are at war?"`
```

Comment:
```suggestion
				`	"Yes. Can you tell me why you are at war?"`
```

Reply:
Considered this one myself, but was uncertain which to go with.